### PR TITLE
feat: card

### PR DESCRIPTION
**Pull request by shashidwivedi**
Card Component:
- Cards with badges
- Cards with dismiss
- Cards with text overlay
- Text only cards
- Vertical/Horizontal cards
- Cards with shadow

image

image

image

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -1 +1,119 @@
...
+.card_horizontal .card-action {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
+
+.card_with-shadow {
+  box-shadow: 0 2px 1px -1px rgb(0 0 0 / 20%), 0 1px 1px 0 rgb(0 0 0 / 14%),
+    0 1px 3px 0 rgb(0 0 0 / 12%);
+}
```
Comment (abhijeetekad) [inline components/card/card.css:119]: You can use variable for this as well

**Review by abhijeetekad — APPROVED** [review]

File: components/card/card.css [hunk 2]
```diff
@@ -1 +1,119 @@
...
+  background-color: var(--color-white);
+  border: 1px solid var(--color-light);
+  display: flex;
+  flex-direction: column;
+  width: calc(var(--space-sm) * 21.875);
+  margin: var(--space-xl) 0;
+}
+
+.card-content {
+  display: flex;
```
Comment (vaibhav-mahalle) [inline components/card/card.css:12]: you can use the utility class here to have better reusability of the code.

File: index.html [hunk 3]
```diff
@@ -387,7 +386,203 @@ <h2>Button</h2>
 
               <button class="btn btn-icon" disabled><i class="fa-solid fa-star"></i></button>
 
-              <button class="btn btn-floating-action" disabled><i class="fa-solid fa-plus"></i></button>
+              <button class="btn btn-floating-action" disabled>
```
Comment (vaibhav-mahalle) [inline index.html:389]: very good and consistent naming.

File: styles/foobar-ui.min.css [hunk 4]
```diff
@@ -1 +1 @@
...
...m) var(--space-xs);font-size:var(--font-size-sm);color:var(--color-dark)}.card-action{display:flex;flex-direction:row;align-items:center;min-height:var(--space-xxl);padding:var(--space-xs);margin-left:auto}.card-btn{background-color:var(--color-white);color:var(--color-dark);height:var(--space-lg);border:1px solid var(--color-light)}.card-btn:hover{box-shadow:none}.card-btn:not(:last-child){margin-right:var(--space-sm)}.card_with-badge{position:relative}.card_text-only,.card_text-overlay{height:min-content}.card_text-overlay .card-text{position:absolute;bottom:0;backdrop-filter:blur(6px) brightness(80%);color:var(--color-white)}.card_text-overlay .card-title{color:unset}.card_text-overlay .card-description{color:unset}.card_text-overlay .card-action{display:none}.card_text-only .card-media{display:none}.card_text-only .card-action{display:none}.card_horizontal{width:calc(var(--space-sm) * 43.75);position:relative}.card_horizontal .card-content{flex-direction:row}.card_horizontal .card-media{width:50%}.card_horizontal .card-action{position:absolute;bottom:0;right:0}.card_with-shadow{box-shadow:0 2px 1px -1px rgb(0 0 0 / 20%),0 1px 1px 0 rgb(0 0 0 / 14%),0 1px 3px 0 rgb(0 0 0 / 12%)}
```
Comment (vaibhav-mahalle) [inline styles/foobar-ui.min.css:1]: please give proper indentation for better readability.

**Review by vaibhav-mahalle — APPROVED** [review]
suggested some minor changes, Please look into it.

File: styles/foobar-ui.min.css (hunk 4 above)
Comment (shashidwivedi) [reply styles/foobar-ui.min.css:1]: Hi Vaibhav, this is minified CSS., that's why indentation is missing

File: components/card/card.css (hunk 2 above)
Comment (shashidwivedi) [reply components/card/card.css:12]: Yes, will be done as part of refactor

File: components/card/card.css (hunk 1 above)
Comment (shashidwivedi) [reply components/card/card.css:119]: Yes, will do as part of refactor later

**Pull request merged**